source/daemon.rst

```diff
@@ -109,6 +93,11 @@ The service supports GET requests with the following paths:
      session and serial. If *reset* is *true*, the *withdrawn* member is not
      present.
 
+In addition, the :command:`/log` endpoint returns :ref:`logging
```

Comment:
Is there a reason this is a prose paragraph instead of part of the list?

Reply:
I was part of the list, but I didn't want repetition of information. So I summarised it in a paragraph and link to the respective sections for further details. Another option was to leave it out altogether, but this felt better.